### DISCUSSION TITLE
Feature Request: Improved User Experience for Team-Only Users

### DISCUSSION THREAD

**Discussion by Amerlander** [umami-software/umami#2964] [Ideas]
I have a few users whose sole purpose is to access websites within a team. Currently, after logging in, they encounter a blank page with the message "No websites available" if they do not have personal websites. This can be confusing and inconvenient for users who are part of a team but do not manage individual websites.

**Feature Suggestions (either of them would work for me):**

**1. Team Buttons on Dashboard:**
Instead of showing "No websites available" when users have no personal websites, provide buttons or links to the teams they are part of. This would allow them to easily switch to the team websites without extra steps.

**2. Auto-Switch to Team Websites:**
Add an option in the user settings to automatically switch to a specific team after login if the user is part of a team and does not have personal websites. This would streamline the experience for team-based users.

**3. Combined View of Personal and Team Websites:**
On the main dashboard, show a combined view of all personal and team websites, allowing users to access everything in one place without needing to switch between teams.

**Benefits:**
- Enhanced user experience for team-only users by making navigation easier.
- Reduced confusion for users without personal websites.
I- ncreased efficiency for users who are part of multiple teams, allowing them to see all relevant websites in one view.

**Comment by makakken** [comment]
i have exactly the same use case! i always have people writing me: "my dashboard is empty" and have to tell them to change to team view. would be really nice if the default dashboard also shows all team-dashboards. 

**Maybe it's possible to let the user choose if he wants to see his own, or a specific teams-dashboard after login inside his profile-settings.**

**Comment by elfensky** [comment]
As a compromise, I would also really like the ability to move websites back from a team to individual users. 

I was testing the team functionality and moved one of my websites to a team, not realising there was no way to move it back.

I've since moved all my sites to the team so it would at least all be in one place, but having to manually go to the team page every so often is getting a bit annoying.
